module-generation/scrape.py

```diff
@@ -0,0 +1,62 @@
+import os
+import json
+import requests
+from github import Github
+
+indexModules = "./scrape-data/"
+os.mkdir(indexModules)
+
+accessToken = os.environ.get('GIT_TOKEN')
+g = Github(accessToken)
+
+user = g.get_user('Terasology')
+repo = user.get_repo("Index")
+content = repo.get_contents("/distros/omega/gradle.properties")
+decodeContent = content.decoded_content.decode()
+for line in decodeContent.split():
+    if line.startswith('extraModules='):
+        modules = line[13:].split(",")
+
+# Collect modules
+for module in modules:
+    repository = user.get_repo(module)
+
+    # Fetch module information
+    try:
+        moduleContent = repository.get_contents("module.txt")
+        decodeModuleContent = moduleContent.decoded_content.decode()
```

Comment:
`decodeModuleContent` is not a reasonable variable name (see my recent comment on method vs variable names)